matrix/matrix.yaml

```diff
@@ -10,20 +10,3 @@
       "do":
         "task": matrix.tasks.health
       "until": health.status.healthy
-- "name": traffic
-  "description": Traffic in the face of Chaos
```

Comment:
Why is it throwing uncaught exceptions?  I'm against a blanket catch because it will mask bugs, but if there are exceptions we know are coming up for reasons that aren't a bug in glitch but that indicate a failure, then we should catch them and turn them in to `TestFailure`s.

I know that traffic isn't implemented yet, but the whole point of adding the gating logic was specifically so we wouldn't have to remove glitch entirely from the default suite; otherwise, we could have just made failures always gating.